{
  "uid" : "bad44cf8a1404c1",
  "name" : "test_index",
  "fullName" : "testSubjectMatter.test_Qsubject_matter.TestOrder#test_index",
  "historyId" : "72bd9b8f4df3b1bd1d8f0bfa4cba5b21",
  "time" : {
    "start" : 1652795834874,
    "stop" : 1652795839198,
    "duration" : 4324
  },
  "status" : "broken",
  "statusMessage" : "AttributeError: module 'pytest' has no attribute 'assume'",
  "statusTrace" : "self = <testcases.testSubjectMatter.test_Qsubject_matter.TestOrder object at 0x00DFBED0>\nsetup_fixture = (<businessPage.subject_matter_page.SubjectMatter object at 0x00DFBF30>, <businessPage.contract_page.Contract object at...7B0>, <businessPage.first_page.First object at 0x00DFB9F0>, <businessPage.optional_page.Optional object at 0x00DFBF90>)\n\n    @pytest.mark.usefixtures('back_fixture')\n    def test_index(self,setup_fixture):\n        subjectmatter, contract, first, optional= setup_fixture\n        for i in range(6):\n            first.to_hq()\n            text = subjectmatter.c_index(i)\n            # K线周期切换\n            data = [\"日\", \"周\", \"月\", \"年\"]\n            for i in data:\n>               contract.cycle(i)\n\ntestSubjectMatter\\test_Qsubject_matter.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <businessPage.contract_page.Contract object at 0x00DFB7B0>, data = '日'\n\n    def cycle(self,data):\n        if data == \"日\":\n            self.get_clickable_element(self.day,\"日K\")\n            time.sleep(1)\n            res = self.get_element_text(self.k_line,\"K线类型日K\")\n>           pytest.assume(res == \"日K\")\nE           AttributeError: module 'pytest' has no attribute 'assume'\n\n..\\businessPage\\contract_page.py:77: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver_fixture",
    "time" : {
      "start" : 1652794360868,
      "stop" : 1652794396581,
      "duration" : 35713
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "setup_fixture",
    "time" : {
      "start" : 1652795833285,
      "stop" : 1652795834874,
      "duration" : 1589
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "login_fixture",
    "time" : {
      "start" : 1652794396581,
      "stop" : 1652794501486,
      "duration" : 104905
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "back_fixture",
    "time" : {
      "start" : 1652795833285,
      "stop" : 1652795833285,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "AttributeError: module 'pytest' has no attribute 'assume'",
    "statusTrace" : "self = <testcases.testSubjectMatter.test_Qsubject_matter.TestOrder object at 0x00DFBED0>\nsetup_fixture = (<businessPage.subject_matter_page.SubjectMatter object at 0x00DFBF30>, <businessPage.contract_page.Contract object at...7B0>, <businessPage.first_page.First object at 0x00DFB9F0>, <businessPage.optional_page.Optional object at 0x00DFBF90>)\n\n    @pytest.mark.usefixtures('back_fixture')\n    def test_index(self,setup_fixture):\n        subjectmatter, contract, first, optional= setup_fixture\n        for i in range(6):\n            first.to_hq()\n            text = subjectmatter.c_index(i)\n            # K线周期切换\n            data = [\"日\", \"周\", \"月\", \"年\"]\n            for i in data:\n>               contract.cycle(i)\n\ntestSubjectMatter\\test_Qsubject_matter.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <businessPage.contract_page.Contract object at 0x00DFB7B0>, data = '日'\n\n    def cycle(self,data):\n        if data == \"日\":\n            self.get_clickable_element(self.day,\"日K\")\n            time.sleep(1)\n            res = self.get_element_text(self.k_line,\"K线类型日K\")\n>           pytest.assume(res == \"日K\")\nE           AttributeError: module 'pytest' has no attribute 'assume'\n\n..\\businessPage\\contract_page.py:77: AttributeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "1ef03d2c7e3ee83e",
      "name" : "log",
      "source" : "1ef03d2c7e3ee83e.txt",
      "type" : "text/plain",
      "size" : 1031
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1
  },
  "afterStages" : [ {
    "name" : "driver_fixture::0",
    "time" : {
      "start" : 1652796124797,
      "stop" : 1652796124797,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  }, {
    "name" : "back_fixture::0",
    "time" : {
      "start" : 1652795839210,
      "stop" : 1652795850500,
      "duration" : 11290
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n\n",
    "statusTrace" : "  File \"D:\\xiaxy\\python\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\xiaxy\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\xiaxy\\automation_test2022.4.22\\testcases\\conftest.py\", line 74, in back_fixture\n    com.back()\n  File \"D:\\xiaxy\\automation_test2022.4.22\\common\\common_fun.py\", line 367, in back\n    self.get_clickable_element(self.Back,\"返回\")\n  File \"D:\\xiaxy\\automation_test2022.4.22\\common\\common_fun.py\", line 276, in get_clickable_element\n    raise e\n  File \"D:\\xiaxy\\automation_test2022.4.22\\common\\common_fun.py\", line 270, in get_clickable_element\n    ele = WebDriverWait(self.driver,timeout,0.5).until(EC.element_to_be_clickable(loc))\n  File \"D:\\xiaxy\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 0
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('back_fixture')"
  }, {
    "name" : "parentSuite",
    "value" : "testSubjectMatter"
  }, {
    "name" : "suite",
    "value" : "test_Qsubject_matter"
  }, {
    "name" : "subSuite",
    "value" : "TestOrder"
  }, {
    "name" : "host",
    "value" : "Win102021AVUCSJ"
  }, {
    "name" : "thread",
    "value" : "10404-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testSubjectMatter.test_Qsubject_matter"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "ab7347877cec1a80",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: 'GET /screenshot' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details",
      "time" : {
        "start" : 1652713072993,
        "stop" : 1652713072993,
        "duration" : 0
      }
    }, {
      "uid" : "5f34871beccb86b7",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: 'GET /screenshot' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details",
      "time" : {
        "start" : 1652712729513,
        "stop" : 1652712729513,
        "duration" : 0
      }
    }, {
      "uid" : "7081071fb768b7b6",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.TimeoutException: Message:",
      "time" : {
        "start" : 1652710825151,
        "stop" : 1652710825151,
        "duration" : 0
      }
    }, {
      "uid" : "3c48065dd41e638",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.TimeoutException: Message:",
      "time" : {
        "start" : 1652710344624,
        "stop" : 1652710344624,
        "duration" : 0
      }
    }, {
      "uid" : "74ed09613eb6b00c",
      "status" : "broken",
      "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20093ms.",
      "time" : {
        "start" : 1652710090199,
        "stop" : 1652710090199,
        "duration" : 0
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "@pytest.mark.usefixtures('back_fixture')" ]
  },
  "source" : "bad44cf8a1404c1.json",
  "parameterValues" : [ ]
}